### PR TITLE
docs(uploader): page content

### PR DESCRIPTION
**Pull request by juliengleyse**
# General summary

# Screenshots

# Check list

-   [ ] (if has style) Add `need: review-integration` label
-   [x] (if has textual documentation) Add `need: review-doc` label
-   [ ] (if has code) Add `need: review-frontend` label
-   [ ] (if has react) Check through the [react dev check list]
-   [ ] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

**Comment by juliengleyse** [comment]
old

File: packages/site-demo/content/product/components/uploader/index.mdx [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # Uploader
 
-## Default
+**Help users select files or medias.**
```
Comment (lionelmoi) [inline packages/site-demo/content/product/components/uploader/index.mdx]:
```suggestion
**Uploaders help users select files or medias.**
```

File: packages/site-demo/content/product/components/uploader/index.mdx [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
 <DemoBlock demo="default" withThemeSwitcher />
 
-## Rounded
+## Variants
```
Comment (lionelmoi) [inline packages/site-demo/content/product/components/uploader/index.mdx]:
```suggestion
## Editable media
```

**Comment by juliengleyse** [comment]
@matmkian c'est bon 👌🏻

**Pull request merged**